build.gradle

```diff
@@ -13,9 +13,10 @@ repositories {
 dependencies {
```

Comment:
It would be great if the build directory and the . dirs weren't merged into main.

Reply:
I tried to fix this and ended up losing some files, I'm not sure how to remove them from my repo. 
❓